packages/@lwc/integration-karma/test/rendering/dynamic-slots/x/lightParent/lightParent.js

```diff
@@ -0,0 +1,19 @@
+import { LightningElement, api } from 'lwc';
+
+export default class LightParent extends LightningElement {
```

Comment:
Additional test cases to consider
1. value is an object created using `Object.create(null)`
2. Value is an object and value bound is a property of the object eg: `slot={foo.bar}`
3. Bound value is for a getter

Reply:
Also:

- null
- boolean `false` (need to check truthiness checks)
- number `0` (ditto)
- empty string
- object


I'd just like to make sure we cover all the bases. If we accidentally introduce a discrepancy between native and synthetic shadow, then that's scary. 😬